server/utils/datePickerUtils.ts

```diff
@@ -4,20 +4,23 @@ import { formatDate } from './utils'
 export const parseDatePickerDate = (datePickerDate: string): Date => {
   const dateFormatPattern = /(\d{1,2})([-/,. ])(\d{1,2})[-/,. ](\d{2,4})/
 
-  if (!dateFormatPattern.test(datePickerDate)) return new Date(NaN)
+  if (!dateFormatPattern.test(datePickerDate)) return null
```

Comment:
Prefer null for invalid values because an invalid date is still technically a "Date".

Reply:
Good idea. That new Date(NaN) came from the prisoner profile version